src/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.Simple.cs

```diff
@@ -0,0 +1,573 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Pipes.Tests
+{
+    /// <summary>
+    /// The Simple NamedPipe tests cover potentially every-day scenarios that are shared
+    /// by all NamedPipes whether they be Server/Client or In/Out/Inout.
+    /// </summary>
+    public abstract class NamedPipeTest_Simple : NamedPipeTestBase
+    {
+        [Theory]
+        [InlineData(PipeOptions.None, PipeOptions.None, false, false)]
+        [InlineData(PipeOptions.None, PipeOptions.None, true, false)]
+        [InlineData(PipeOptions.None, PipeOptions.None, false, true)]
+        [InlineData(PipeOptions.None, PipeOptions.None, true, true)]
+        [InlineData(PipeOptions.None, PipeOptions.Asynchronous, false, false)]
+        [InlineData(PipeOptions.None, PipeOptions.Asynchronous, true, false)]
+        [InlineData(PipeOptions.None, PipeOptions.Asynchronous, false, true)]
+        [InlineData(PipeOptions.None, PipeOptions.Asynchronous, true, true)]
+        [InlineData(PipeOptions.Asynchronous, PipeOptions.None, false, false)]
+        [InlineData(PipeOptions.Asynchronous, PipeOptions.None, true, false)]
+        [InlineData(PipeOptions.Asynchronous, PipeOptions.None, false, true)]
+        [InlineData(PipeOptions.Asynchronous, PipeOptions.None, true, true)]
+        [InlineData(PipeOptions.Asynchronous, PipeOptions.Asynchronous, false, false)]
+        [InlineData(PipeOptions.Asynchronous, PipeOptions.Asynchronous, true, false)]
+        [InlineData(PipeOptions.Asynchronous, PipeOptions.Asynchronous, false, true)]
+        [InlineData(PipeOptions.Asynchronous, PipeOptions.Asynchronous, true, true)]
```

Comment:
This looks like it's every combination of each of the four binary values?  If so, you might consider making that more explicit via MemberData instead of InlineData, e.g.
```C#
public static IEnumerable<object[]> OneWayReadWritesMemberData()
{
    var options = new [] { PipeOptions.None, PipeOptions.Asynchronous };
    var bools = new [] { false, true };
    foreach (PipeOption serverOption in options)
    foreach (PipeOption clientOption in options)
    foreach (bool asyncServerOps in bools)
    foreach (bool asyncClientOps in bools)
        yield return new object[] { serverOption, clientOption, asyncServerOps, asyncClientOps };
}
```